packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -30,7 +30,7 @@ describe("Runtime", () => {
                         return {
                             deltaManager: new MockDeltaManager(),
                             quorum: new MockQuorum(),
-                            logger: new MockLogger(),
+                            taggedLogger: new TaggedLoggerAdapter(new MockLogger()),
```

Comment:
You don't need to use the TaggedLoggerAdapter here. It only exists for super old back-compat.

Reply:
Sounds good, wasn't sure since both `logger` and `taggedLogger` share the same type if there was perhaps some implicit API contract that MockLogger would fail to fulfill, so was doing this out of caution.